pkgs/top-level/python-packages.nix

```diff
@@ -1877,6 +1879,18 @@ in {
 
   gmpy2 = callPackage ../development/python-modules/gmpy2 { };
 
+  # alpha release, big refactor, adds cython support
+  # see https://github.com/aleaxit/gmpy/issues/146, https://github.com/aleaxit/gmpy/issues/199
+  gmpy2_2_1 = (callPackage ../development/python-modules/gmpy2 {}).overridePythonAttrs (oldAttrs: rec {
```

Comment:
We can't do that for Python packages because having different versions of the same package in `$PYTHONPATH` is not supported. You could upgrade the default gmpy2 instead.

Reply:
However, I don't see a specific version requirement for gmpy2 in pplpy's setup.py.

Reply:
Their [readme](https://pypi.org/project/pplpy/) specifies

> gmpy2: version 2.1.0a4 from sources

There is a beta release coming probably soonish (https://github.com/aleaxit/gmpy/issues/199), so maybe upgrading after that makes sense.

(I meant to ping you in the PR, not domenkozar. Damn autocomplete)